docs/UserGuide.md

```diff
@@ -162,11 +163,15 @@ ____________________________________________________________
 
 #### 2.2.2. Add an item: `items add`
 
-Add an item to the system.
+Are you sometimes overwhelmed by handling a large variety of items? No worries, (items add) feature can help you.
```

Comment:
maybe do step by step instead of talking in story form, not too sure, we can discuss again